resources/lib/resumepoints.py

```diff
@@ -84,7 +84,7 @@ def set_resumepoint(self, uuid, title, url, watch_later=None, position=0, total=
             payload = dict(position=position, total=total, url=url)
 
         if watch_later is not None:
-            payload['watchLater'] = watch_later
+            payload['watchLater'] = str(watch_later).lower()
```

Comment:
Are you sure this is correct? The JSON payload we get back from: https://video-user-data.vrt.be/resume_points are proper JSON booleans, not strings.

Reply:
I can also confirm that in the request, true booleans are being used.

Reply:
I'm not sure, I will remove this again.